.github/workflows/rebuild-protos.yml

```diff
@@ -0,0 +1,30 @@
+name: Rebuild Protos
+
+on:
```

Comment:
you could test this by add pull_request: here and changing some proto file. Then we just close the opened PR

Reply:
In theory it shouldn't make a PR if the protos are up-to-date, but I can try adding that temporarily to at least run the workflow

Reply:
I believe this error is due to the checked out branch not having the same commit hash as `HEAD`:

https://github.com/cosmos/cosmos-rust/pull/24/checks?check_run_id=2231232423

```
Checking the base repository state
  /usr/bin/git symbolic-ref HEAD --short
  fatal: ref HEAD is not a symbolic ref
  /usr/bin/git rev-parse HEAD
  2e955eca0b443bc9480d277281bf893d1ffa3420
  Working base is commit '2e955eca0b443bc9480d277281bf893d1ffa3420'
  Error: When the repository is checked out on a commit instead of a branch, the 'base' input must be supplied.
```

So I think the only way to actually test it is to merge it unfortunately